.travis.yml

```diff
@@ -51,6 +51,24 @@ env:
 
 matrix:
   exclude:
+    # Lack of required C++11 level. Default uses only c++0x
+    - os: linux
+      env: PROJECT_DIR=examples/msgpack TOOLCHAIN=default
+
+    # I'dont know what is the problem, build fails with error messages:
+    # clang: error: argument unused during compilation: '-pie'
```

Comment:
Yes. This is default:
https://travis-ci.org/ingenue/hunter/jobs/127172996
and this is the clang "error":
https://travis-ci.org/ingenue/hunter/jobs/127173004